### PR TITLE
Add no home option for adding ceilometer user to libvrt

### PR DESCRIPTION
**Pull request by edtubillara**

### CODE REVIEW

**Comment by nirajdp76** [comment]
ok to test

File: roles/ceilometer-data/tasks/main.yml [hunk 1]
```diff
@@ -38,6 +38,7 @@
     name: ceilometer
     groups: "{{ ceilometer.libvirt_group[ursula_os] }}"
     append: yes
+    createhome: no
```
Comment (nirajdp76) [inline roles/ceilometer-data/tasks/main.yml]: need to also add
`home=/nonexistent`

**Comment by nirajdp76** [comment]
ok to test

**Comment by jeffreyzhou** [comment]
retest

**Comment by jeffreyzhou** [comment]
retest

**Pull request merged**